plugin_template/templates/plugin_template/plugin_template.html

```diff
@@ -0,0 +1,6 @@
+{% extends "kolibri/base.html" %}
+{% load styleguide_tags %}
```

Comment:
better call this plugintemplate_tags, and rename styleguide_tags.py accordingly.